bot/cogs/bot.py

```diff
@@ -60,6 +62,20 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:  # more than three lines
+            try:
+                tree = ast.parse(msg.content)
+
+                # Attempts to parse the message into an AST node.
+                # Invalid Python code will raise a SyntaxError.
+                # => text which is not Python code will be filtered.
+                if not all(isinstance(node, ast.Expr) for node in tree.body):  # we dont want hi\nthere\nguys\nD
```

Comment:
`# we dont want hi\nthere\nguys\nD` is what I meant. Try removing it and adding a new comment underneath

Reply:
Something like:
`# Single-line words are interpreted as expressions.`
`# This checks if all nodes were parsed as expressions.`